bundles/org.openhab.core/src/test/java/org/openhab/core/library/unit/SmartHomeUnitsTest.java

```diff
@@ -168,6 +168,40 @@ public void testKnotUnitSymbol() {
         assertThat(SmartHomeUnits.KNOT.getSymbol(), is("kn"));
         assertThat(SmartHomeUnits.KNOT.toString(), is("kn"));
     }
+    
+    @Test
+    public void testVarUnitSymbol() {
+        assertThat(SmartHomeUnits.VAR.getSymbol(), is("var"));
+        assertThat(SmartHomeUnits.VAR_HOUR.toString(), is("varh"));
```

Comment:
Can you align the unit test to the others? One assertion for `getSymbol()` and one assertion for `toString()` for each unit.

```
        assertThat(SmartHomeUnits.VAR.getSymbol(), is("var"));
        assertThat(SmartHomeUnits.VAR.toString(), is("var"));
        assertThat(SmartHomeUnits.VAR_HOUR.getSymbol(), is("varh"));
        assertThat(SmartHomeUnits.VAR_HOUR.toString(), is("varh"));
```

Additionally dose it make sense to add a test for the comparison to other units of the dimension power (e.g. (`1 var = 1 W`)?

Reply:
There is a problem with this check:
`assertThat(SmartHomeUnits.VAR_HOUR.getSymbol(), is("varh"));`

Namely, that ProductUnit does not calculate the symbol propert when there are multiple units. I'm not sure why is it so, but there's a fixme in the sourcecode: https://github.com/unitsofmeasurement/uom-se/blob/master/src/main/java/tec/uom/se/unit/ProductUnit.java#L111

> Additionally dose it make sense to add a test for the comparison to other units of the dimension power (e.g. (1 var = 1 W)?
I'm uncertain about that. Wikipedia sais: "The volt-ampere is _dimensionally_ _equivalent_ to the watt (in SI units, 1 VA = 1 N m A-1 s -1 A = 1 N m s -1 = 1 J s -1 = 1 W)." But as I understand there's no such thing that we convert from var to watts because you will measure different things in W/VA/var. See: https://en.wikipedia.org/wiki/Volt-ampere

How should such a test look like in your opinion?

Reply:
note: I've pushed a version where I've entered the getSymbol() checks, but commented them out, because they now break the tests.

Reply:
I did my research on German [Wikipedia](https://de.m.wikipedia.org/wiki/Var_(Einheit)). I have the feeling there is a mistake about the formular on it compared to your link. Thus it does not make sense to introduce further tests.

Reply:
> > ProductUnit does not calculate the symbol
> 
> We can open an issue in https://github.com/unitsofmeasurement/uom-se/issues and ask them to fix it .

Did some research and it looks like the do not have the intention to fix this: https://github.com/unitsofmeasurement/uom-se/issues/184
The preferred way is to use the toString() unless dealing with a specific BaseUnit.

Reply:
As I understand now the symbol property means that a unit has a symbol directly attached to it. For derived units this should be null, and toString should implement the display of the derived symbol.
The test complying with this convertion should look like this:

`assertNull(SmartHomeUnits.VAR_HOUR.getSymbol());

@cweitkamp  please let me know if you would like me to change the tests this way.

Reply:
Okay, I see. Thanks for the research. We should keep the `toString()` tests and omit the `getSymbol()` tests for those units.

Reply:
Thank you, updated the code.